test/src/gatherers/screenshots.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright 2016 Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+/* eslint-env mocha */
+
+const ScreenshotsGather = require('../../../src/gatherers/screenshots');
+const assert = require('assert');
+let screenshotsGather = new ScreenshotsGather();
+
+describe('Screenshot gatherer', () => {
+  it('returns an artifact for a real trace', () => {
+    const traceData = require('../audits/performance/progressive-app.json');
+    return screenshotsGather.postProfiling(undefined, {traceContents: traceData}).then(_ => {
```

Comment:
unfortunate to bake in assumptions about what method(s) the screenshots gather needs called to produce an artifact, but we really need a "run scheduler with this gatherer, this mocked driver, and this trace" test class to do that right